kernel/pci/src/lib.rs

```diff
@@ -420,7 +421,7 @@ impl PciDevice {
 
         self.pci_write(bar_offset, 0xFFFF_FFFF);          // Step 1
         let mut mem_size = self.pci_read_32(bar_offset);  // Step 2
-        mem_size.set_bits(0..4, 0);                       // Step 3
+        mem_size &= 0xFFFFFFF0;                           // Step 3
```

Comment:
is this better? I don't necessarily think it's any clearer

Reply:
this is temporary since I wanted to remove the bitfield crate and this was the only place left